scipy/sparse/csgraph/_laplacian.py

```diff
@@ -29,6 +31,10 @@ def laplacian(csgraph, normed=False, return_diag=False, use_out_degree=False):
         If True, then use out-degree instead of in-degree.
         This distinction matters only if the graph is asymmetric.
         Default: False.
```

Comment:
Just seeing that we could add the defaults for `normed` and `return_diag`. Would be great if you could do this, I cannot do a suggestion as it's too far above.

Reply:
done